src/assets/css/style.css

```diff
@@ -7,3 +7,84 @@ body {
   background: linear-gradient(to right, yellow, red)
 }
 
+
+*{
+  padding: 0;
+  margin: 0;
+}
+/* 
+body{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 100px;
+  background-color: rgb(5, 5, 5);
+} */
+
+
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 32px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+.slider {
```

Comment:
How do we recognize the one for light or dark theme.... please restructure your code again

Reply:
there is no light or dark theme in the figma, its the same, just the color changes when toggle checked and unchecked